json-guides/microprofile-config.json

```diff
@@ -31,25 +31,53 @@
             ]
         },
         {
-            "name": "AddLibertyMPFaultTolerance",
-            "title": "Enabling MicroProfile Config",
+            "name": "EnableMPConfig",
+            "title": "Enabling the MicroProfile Config feature",
             "description": [
-                "Begin by enabling the MicroProfile Config feature in your <code>server.xml</code> file."
+                "Begin by enabling the MicroProfile Config 1.1 feature in your <code>server.xml</code> file."
             ],
             "instruction": [
-              "Add <b>&lt;feature>mpConfig-1.1&lt;/feature>."
+              "Add the following element declaration inside the featureManager element to the <code>server.xml</code> file, or click <action tabindex='0' role='button' title='Enable MicroProfile Config' aria-label='Enable MicroProfile Config' onkeypress=\"microprofileConfigCallBack.addMicroProfileConfigFeatureButton(event)\" onclick=\"microprofileConfigCallBack.addMicroProfileConfigFeatureButton(event)\"><b>&lt;feature&gtmpConfig-1.1&lt;/feature&gt;</b></action>. Then, click <action tabindex='0' role='button' title='Save' aria-label='Save' onkeypress=\"microprofileConfigCallBack.saveServerXMLButton(event)\" onclick=\"microprofileConfigCallBack.saveServerXMLButton(event)\"><b>Save</b></action> on the editor menu pane."            ],
+            "content":[
+              {
+                "displayType":"fileEditor",
+                "fileName": "server.xml",
+                "preload": [
+                  "<?xml version=\"1.0\"?>",
+                  "<server description=\"Sample Liberty server\">",
+                  "   <featureManager>",
+                  "      <feature>cdi-1.2</feature>",
+                  "      <feature>jaxrs-2.0</feature>",
+                  "   </featureManager>",
+                  "   <httpEndpoint host=\"*\" id=\"defaultHttpEndpoint\" httpsPort=\"${default.https.port}\"",
+                  "   httpPort=\"{default.http.port}\"/>",
+                  "</server>"
+                ],
+                "readonly": [
+                  {
+                      "from": "1",
+                      "to": "5"
+                  },
+                  {
+                      "from": "6",
+                      "to": "9"
```

Comment:
Can the user actually type in anything as you're marking lines 1 thru 9 readonly?